services/x-net/setup_rpc_compose.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+cat > /subspace/docker-compose.yml << EOF
+version: "3.7"
+
+volumes:
+  caddy_data: {}
+  archival_node_data: {}
+
+services:
+  # caddy reverse proxy with automatic tls management using let encrypt
+  caddy:
```

Comment:
Maybe it makes sense to add log rotation here `    logging:
      driver: "json-file"
      options:
        max-size: "50m"
        max-file: "10"`

Reply:
There are some default limits in Docker by default, 250M IIRC, but I don't think it keeps more than 1 file. Log rotation shouldn't be necessary though if we have DataDog for this.